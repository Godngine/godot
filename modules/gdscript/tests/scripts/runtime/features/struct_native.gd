var test_struct: PropertyInfo


func test():
    print(test_struct)
    print(test_struct.name)
    test_struct.name = "test"
    print(test_struct.name)
