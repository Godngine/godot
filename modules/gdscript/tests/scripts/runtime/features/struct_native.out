GDTEST_OK
[name: "", class_name: &"", type: 0, hint: 0, hint_string: "", usage: 6]

test
